### ISSUE TITLE
Currency change

### ISSUE THREAD

**Issue by ayimdomnic** [majid-n/printkala#1]
Hello, I appreciate the good work you''re doing on this repo, However am having issues changing the price naming method to English, How can I go about it?

**Comment by majid-n** [comment]
Hello dear @ayimdomnic, you need to change it in 
views/welcome.blade.php line 33 :
<span>{{ number_format($product->price) . ' ریال' }}</span>
and  views/cart.blade.php line19 , 28 :
<td class="hidden-xs" align="left" class="itemTotal">{{ number_format($item->total). ' ریال' }}</td>
&
<td align="left" class="sumTd">{{ number_format($total) . ' ریال' }}</td>

of course you can create a new variable somewhere for example in the config/app.php file to make it easy to change.
hope it helps

**Comment by ayimdomnic** [comment]
thanks av made a global variable though most of the symbols are still in persian.. Like when adding a new product and all.. the user Test on cart doesn't work as well

**Comment by majid-n** [comment]
yeah i'm still working on this project and i will push new changes and bug fixes today. if you need help just let me know.

**Comment by ayimdomnic** [comment]
okay thanks
